src/librustc_codegen_llvm/abi.rs

```diff
@@ -481,6 +490,8 @@ impl<'tcx> FnAbiLlvmExt<'tcx> for FnAbi<'tcx, Ty<'tcx>> {
         if cconv != llvm::CCallConv {
             llvm::SetInstructionCallConv(callsite, cconv);
         }
+
+        // FIXME(wesleywiser, eddyb): We should apply `nounwind` as appropriate to this callsite.
```

Comment:
Can you move this to the top of the function? That's where it is in the other one.

Reply:
Could also mention `noreturn` since I believe they're in a similar situation.